consts/src/lib.rs

```diff
@@ -377,9 +377,9 @@ mod helpers {
         ))
     }
 
-    pub fn get_id_cred<'a>(cred: &'a [u8]) -> BytesIdCred {
-        let (_g, kid) = parse_cred(cred).unwrap();
-        [0xa1, 0x04, 0x41, kid]
+    pub fn get_id_cred<'a>(cred: &'a [u8]) -> Result<BytesIdCred, EDHOCError> {
+        let (_g, kid) = parse_cred(cred)?;
```

Comment:
This early return could be easily replaced with a `.map(|(_g, kid)| [0xa1, 0x04, 0x41, kid])`.

Reply:
Thanks, will apply it but in #150.